datastore/gcloud/aio/datastore/filter.py

```diff
@@ -0,0 +1,111 @@
+from typing import Any
+from typing import Dict
+from typing import List
+
+from gcloud.aio.datastore.constants import CompositeFilterOperator
+from gcloud.aio.datastore.constants import PropertyFilterOperator
+from gcloud.aio.datastore.value import Value
+
+
+class _BaseFilter:
+    json_key = ''
```

Comment:
This can just be `json_key: str` to be abstract rather than giving it the default value.

Reply:
Nice!